src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('Roboto-Medium.ttf');
+}
+
+html {
+  --brand-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+header {
```

Comment:
Use class selectors instead of tag. Fix in all cases.

Reply:
I have changed tag selectors in favour of classes.